engine/schema/src/main/resources/META-INF/db/schema-41610to41700.sql

```diff
@@ -272,6 +291,7 @@ CREATE VIEW `cloud`.`volume_view` AS
         disk_offering.display_text disk_offering_display_text,
         disk_offering.use_local_storage,
         service_offering.system_use,
+        disk_offering.system_use,
```

Comment:
Hi @DK101010 the build is failing because this column does not exist:

````
Caused by: java.sql.SQLSyntaxErrorException: Unknown column 'disk_offering.system_use' in 'field list'
        at com.cloud.utils.db.ScriptRunner.runScript(ScriptRunner.java:185)
        at com.cloud.utils.db.ScriptRunner.runScript(ScriptRunner.java:87)
        at com.cloud.upgrade.DatabaseUpgradeChecker.runScript(DatabaseUpgradeChecker.java:215)
        ... 52 more
2022-04-07 13:39:12,242 DEBUG [c.c.u.d.T.Transaction] (main:null) (logid:) Rolling back the transaction: Time = 369 Name =  Upgrade; called by -TransactionLegacy.rollback:888-TransactionLegacy.removeUpTo:831-TransactionLegacy.close:655-DatabaseUpgradeChecker.upgrade:314-DatabaseUpgradeChecker.check:389-CloudStackExtendedLifeCycle.checkIntegrity:64-CloudStackExtendedLifeCycle.start:54-DefaultLifecycleProcessor.doStart:178-DefaultLifecycleProcessor.access$200:54-DefaultLifecycleProcessor$LifecycleGroup.start:356-Iterable.forEach:75-DefaultLifecycleProcessor.startBeans:155
````

Reply:
Hi @nvazquez, is deleted. Is cames not from me, I guess it was a little mistake during the merge.

Reply:
@DK101010 ok, I see there are conflicts again, sorry will need further fixing :)

Reply:
@nvazquez ok :D